mackup/applications/fisherman.cfg

```diff
@@ -0,0 +1,6 @@
+[application]
+name = Fisherman
+
+[configuration_files]
+.config/fisherman
```

Comment:
I believe that this folder shouldn't be synced because may contain git files. `~/.config/fish/fishfile` is all you need: you just run `fisherman` without subcommands and it installs all your packages from the fishfile.

@jbucaran correct me if I'm wrong.